tools/cold-store/src/cli.rs

```diff
@@ -143,6 +174,82 @@ fn copy_next_block(store: &NodeStorage, config: &NearConfig, hot_runtime: &Arc<N
         .expect(&std::format!("Failed to update cold HEAD to {}", next_height));
 }
 
+fn copy_all_blocks(store: &NodeStorage, max_mem_size: usize, min_batch_size: usize, check: bool) {
+    copy_all_data_to_cold(
+        (*store.cold_db().unwrap()).clone(),
+        &store.get_store(Temperature::Hot),
+        max_mem_size,
+        min_batch_size,
+    )
+    .expect("Failed to do migration to cold db");
+
+    // If FINAL_HEAD is not set for hot storage though, we default it to 0.
+    let hot_final_head = store
+        .get_store(Temperature::Hot)
+        .get_ser::<Tip>(DBCol::BlockMisc, FINAL_HEAD_KEY)
+        .unwrap_or_else(|e| panic!("Error reading hot FINAL_HEAD: {:#}", e))
+        .map(|t| t.height)
+        .unwrap_or(0);
+
+    update_cold_head(
+        &*store.cold_db().unwrap(),
+        &store.get_store(Temperature::Hot),
+        &hot_final_head,
+    )
+    .expect(&std::format!("Failed to update cold HEAD to {}", hot_final_head));
+
+    if check {
+        for col in DBCol::iter() {
+            if col.is_cold() {
+                println!(
+                    "Performed {} {:?} checks",
+                    check_iter(
+                        &store.get_store(Temperature::Hot),
+                        &store.get_store(Temperature::Cold),
+                        col,
+                        &vec![],
+                    ),
+                    col
+                );
+            }
+        }
+    }
+}
+
+fn check_key(
+    first_store: &near_store::Store,
+    second_store: &near_store::Store,
+    col: DBCol,
+    key: &[u8],
+) {
+    let first_res = first_store.get(col, key);
+    let second_res = second_store.get(col, key);
+
+    assert_eq!(first_res.unwrap(), second_res.unwrap());
+}
+
+fn check_iter(
+    first_store: &near_store::Store,
+    second_store: &near_store::Store,
+    col: DBCol,
+    no_check_rules: &Vec<Box<dyn Fn(DBCol, &Box<[u8]>) -> bool>>,
```

Comment:
Why do you need this? Can you add a comment explaining that?

Reply:
Yeah, we only need that when testing correctness after copying block by block, so can delete it here.